model-optimizer/extensions/ops/ctc_greedy_decoder_seq_len.py

```diff
@@ -0,0 +1,93 @@
+"""
+ Copyright (C) 2017-2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.extractor import bool_to_str
+from mo.graph.graph import Node, Graph
+from mo.middle.passes.convert_data_type import np_data_type_to_destination_type
+from mo.ops.op import Op
+from mo.utils.error import Error
+
+
+class CTCGreedyDecoderSeqLenOp(Op):
```

Comment:
Loading of the model with this operation with MO IR Reader will fail (you must update the MO IR Reader to make it work). But you marked the test as passed in the PR description. Please, explain.

Reply:
just mistake. MO IR Reader was updated and works correct now.